src/auth0_fastapi/server/routes.py

```diff
@@ -123,7 +133,30 @@ async def backchannel_logout(
                 raise HTTPException(status_code=400, detail=str(e))
             return Response(status_code=204)
 
+    if config.mount_connected_account_routes:
+        @router.get("/auth/connect-account")
```

Comment:
For symmetry, this should be the same as the nextjs-auth0 [implementation](https://github.com/auth0/nextjs-auth0/blob/b44aa978deacd51fed1c68d8e44c4070d55cc7f7/src/server/client.ts#L442):
```suggestion
        @router.get("/auth/connect")
```
This makes it easier for the front-end library to behave the same no matter what back-end framework+sdk is used.

Reply:
oh, I see that `/auth/connect` is already used for account linking 😬 I think that the account-linking approach should be discontinued (to be confirmed based on the deprecation plain). Is it intended to keep it as-is for now?

(if that's the case, I might need to override the url used by the front-end... We already have a way to do it, just a bit more complex setup for our customers)

Reply:
Yeh would have preferred `/auth/connect` but its taken. Would love to remove it but its a breaking change so think I had to avoid it. The other option would be to make `mount_connect_routes` and `mount_connected_account_routes ` mutually exclusive, so the meaning of the `/auth/connect` route is dependent on which config setting you have enabled.